src/results/financial.jl

```diff
@@ -338,7 +338,7 @@ function get_depreciation_schedule(p::REoptInputs, tech::Union{AbstractTech,Abst
 
     federal_itc_fraction = 0.0
     try 
-        # TODO add Hot/ColdThermalStorage.total_itc_fraction to struct; currently only in ElectricStorage
+        # TODO add Hot/ColdThermalStorage.total_itc_fraction to struct; currently only in ElectricStorage 
```

Comment:
@Bill-Becker it looks like you added this comment but these techs do have total_itc_fraction. Can we remove this TODO or is it referencing something else that I'm missing?

Reply:
done https://github.com/NREL/REopt.jl/pull/508/commits/279ec116e454be98f2f35433bf8ded2694fff2aa